### DISCUSSION TITLE
[Projects Beta] Add multiple repos per view

### DISCUSSION THREAD

**Discussion by seafarer** [community/community#6717] [Projects and Issues] [answered]
We used to be able to add multiple repos per view, but last week this functionality stopped working. Now I don't know how to create a view that brings in issues from multiple repos. 

The filter query we used before:
```
repo:"org/repo-1, org/repo-2, org/repo-3"
```
This would bring in issues from all three repos, which worked great. Our org has 25+ active repos, but many are similarly related, so being able to create views of 2-4 repos is ideal for our workflow.

**Comment by azenMatt** [comment, marked as answer]
Hi @seafarer - sorry about this, there was a bug fix we just applied that would have impacted this. 

Because the `,` can be used to create an OR search, the correct way to do this would be:

`repo:org/repo1,org/repo2,org/repo3`

If your search term has spaces in it, the you would wrap the individual terms in `"`

`label:"example label one","example label two"`

Previously, we were not correctly escaping the `,` character when it was wrapped in `"`. Which meant that the query like you had above was possible, but if you wanted to find a sting that had an `,` in it - then that would be impossible. (for example, `label:"label name with a , in it"` would not create a filter correctly)

**Reply by seafarer** [reply]
Awesome thanks for the response! Working now.
